yaml/csr.yaml.m4

```diff
@@ -117,6 +122,93 @@
       msb: 31
       lsb: 0
 
+ifelse(eval(ZICNTR != 0), 1, [[[
+- csr: cycle
+  description: >
+    Unprivileged alias of lower 32 Machine Cycle Counter
+  address: 0xC00
+  privilege_mode: M
```

Comment:
Shall this (and other CSRs marked with "U--" in the user manual really have privilege_mode:M in this file?

Reply:
They should be `U`, thanks. I've pushed an update to change them now.